Tests/FluentAssertions.Shared.Specs/GenericCollectionAssertionsSpecs.cs

```diff
@@ -34,6 +34,444 @@ public void When_asserting_equality_with_a_collection_built_from_params_argument
             act.ShouldNotThrow();
         }
 
+        #region BeEquivalentTo
+
+        [TestMethod]
+        public void Check_asserting_equivallency_by_assert_method_should_check_it_regardless_of_the_order()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            //Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            List<DateTime> actualDates = new List<DateTime>
+            {
+                DateTime.Today,
```

Comment:
Please use constant values in tests. It seems that this test can fail on the edge of the day.
In general, using non-deterministic data in tests is a code smell.